<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemove">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>300</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="actionImportCSV"/>
    <addaction name="actionExportCSV"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportCSV">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/img/import.png</normaloff>:/new/prefix1/img/import.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/img/export.png</normaloff>:/new/prefix1/img/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/img/exit.png</normaloff>:/new/prefix1/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
